src/blend_mesh.rs

```diff
@@ -37,14 +38,7 @@ pub fn blend_mesh_loader(
             ..Default::default()
         };
 
-        commands.entity(entity).insert_bundle((
-            bundle.mesh,
-            bundle.material,
-            bundle.main_pass,
-            bundle.draw,
-            bundle.visible,
-            bundle.render_pipelines,
-        ));
+        commands.entity(entity).insert_bundle(bundle);
```

Comment:
For me this made the meshes disappear. I suspect it is because the PbrBundle includes transform and global_transform. That's why the old version manually inserted a selection of the bundle.

This may be fixed in 0.6, but if your meshes aren't displaying you can try restricting what is inserted here.